examples/extras/classification-confusion-matrix.ipynb

```diff
@@ -0,0 +1,289 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "47170397",
+   "metadata": {},
+   "source": [
+    "# Classification Confusion matrix\n",
+    "\n",
+    "This notebook plots a confusion matrix for a multi-class, single label classification.\n",
+    "\n",
+    "It assumes that your project has a number of labels from multiple labelers, where the Annotation Type is Radio."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "639ea036",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2022-02-24T00:28:45.968825Z",
+     "start_time": "2022-02-24T00:28:45.949740Z"
+    },
+    "code_folding": [
+     0
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Imports and etc\n",
+    "\n",
+    "from sklearn.metrics import ConfusionMatrixDisplay\n",
```

Comment:
Maybe before the imports we also have an Install section similar to this notebook:
https://github.com/Labelbox/labelbox-python/blob/develop/examples/annotation_types/basics.ipynb

I think we are also trying to turn all installs into a one liner and also with -q to keep it simple

Otherwise lgtm!

Reply:
Good point, just added install, links to github and colab at the top and some headers.